### PR TITLE
JS example for REST interface.

### PR DESCRIPTION
**Pull request by gsaluja9**
Works with the latest webui and lets user AddImage

### CODE REVIEW

File: examples/rest_api/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <body onload="run_requests()">
+        <pre id="output"></pre>
+        <h6>AddImage. Select a file to continue</h6>
+        <form id="addimage" enctype="multipart/form-data">
+            <input id="fileupload" name="blob" type="file" />
+            <input type="submit" value="submit" id="submit" />
+
+        </form>
+    </body>
+</html>
```
Comment (luisremis) [inline examples/rest_api/index.html:15]:
```suggestion
</html>

```

File: examples/rest_api/rest_api.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        displayContent(response);
+    }, sessionToken = sessionStorage.getItem("session_token"));
+    
+}
+
+window.addEventListener("load", (event)=>{
+    console.log("hello world");
+    const form = document.getElementById("addimage");
+    form.addEventListener('submit', addImage);
+})
```
Comment (luisremis) [inline examples/rest_api/rest_api.js:120]:
```suggestion
})

```

**Review by luisremis — COMMENTED** [review]
great! Just need to add the AddImage example.

**Comment by gsaluja9** [comment]
> 



> great! Just need to add the AddImage example.

it's already in.

**Review by luisremis — DISMISSED** [review]
works on my system. good to go!

**Review by luisremis — DISMISSED** [review]

**Review by luisremis — APPROVED** [review]

**Pull request merged**